test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_String.java

```diff
@@ -1377,6 +1405,34 @@ public void test_toUpperCase() {
 
 		AssertJUnit.assertTrue("Wrong conversion", "\u00df".toUpperCase().equals("SS"));
 
+		// Long strings and strings of lengths near multiples of 16 bytes to test vector HW acceleration edge cases.
+		AssertJUnit.assertTrue("toUpperCase case conversion failed on a long string",
```

Comment:
Same here. `org.testng.Assert` should be used.

Reply:
09078d4